docs/tutorial.rst

```diff
@@ -204,41 +206,82 @@ b) Writing a gain calibration file.
       dd = []
       fl = []
       ch = []
-      for ant in antenna_numbers:
+      for ant in ant_array:
           dd.append(gains[pol][ant])
```

Comment:
I think that would be fine. I didn't actually write this, I just tweaked what @zakiali had, but if he's on board we can significantly simplify it.